### ISSUE TITLE
Adding Takeoff Roll Pilot Not Flying (PNF) callouts

### ISSUE THREAD

**Issue by FelixFeltersnatch** [invernyx/md11-bugs#598]
### Is your feature request related to a problem? Please describe.

It is not. Other than that the add-on doesn't have them.

### Describe the solution you'd like

During the takeoff roll, there are several callouts that the PNF makes that a critical to maintaining good Situational Awareness and are also proper CRM, regardless of the particular SOP.

These are:

"Eighty Knots" (80kts) - this callout is used as a speed check, an incapacitation check, and/or a takeoff roll regime delineation depending on a particular shop's SOPs. However, the callout is ubiquitous

"Vee One" (V1) - this is the minimum decision speed, the speed at which once passed, you are legally committed to a takeoff regardless of any subsequent failure (unless the aircraft literally won't fly, see AAY483)

"Rotate" (Vr) - this call is to let the PF know to start pulling back on the column

"Vee Two" (V2) - this callout is to know that the engine-out climb speed has been reached. 

All of these callouts are made by the PNF (pilot not flying)

I would also like to add that I would be more than happy to record these callouts.

### Describe alternatives you've considered

None.

### Additional context

These callouts can be found as standard on many high fidelity add-ons, including add-ons like the PMDG 737NG series.

**Comment by GR1MMly** [comment]
I agree this would be a nice convenience to add on. The specific SOPs im familiar with have the first call limited to "80 knots" while the thrust and clamp are silently verified during that callout.
